examples/seismic/skew_self_adjoint/wavesolver.py

```diff
@@ -0,0 +1,310 @@
+from devito import Function, TimeFunction
+from examples.seismic import PointSource, Receiver
+from examples.seismic.skew_self_adjoint.utils import *
+from examples.seismic.skew_self_adjoint.operators import *
+
+
+class SSA_ISO_AcousticWaveSolver(object):
+    """
+    Solver object for a scalar isotropic variable density visco- acoustic skew
+    self adjoint wave equation that provides operators for seismic inversion problems
+    and encapsulates the time and space discretization for a given problem setup.
+
+    Parameters
+    ----------
+    npad : int, required
+        Number of points in the absorbing boundary
```

Comment:
full stops

Reply:
not sure what mean by full stops? perids at end of sentences?

Reply:
pretty sure you mean periods ...